### PR TITLE
column widths changes

### PR DESCRIPTION
**Pull request by jtrein**
handles #39 

* sets width of `code` to `125px`
* updates function to handle any number of fixed widths and divides the rest up; if no fixed widths, divides evenly.

### CODE REVIEW

**Comment by Chris-Petty** [comment]
protip: there are [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) that close related issues when the PR is merged

Closes #39

File: src/js/components/Search.jsx [hunk 1]
```diff
@@ -35,17 +35,33 @@ export class Search extends PureComponent {
...
-    for (let i = 0; i < columnWidths.length; i += 1) {
-      columnWidths[i] = tableWrapWidthDivided;
+  handleCalculateColumnWidths(columns, tableWidth, columnFixedWidths = {}) {
+    if (columns === undefined || tableWidth === undefined) return null;
+    const columnWidths = [];
+
+    let totalFixedWidth = 0;
+    const fixedWidthArray = Object.values(columnFixedWidths);
+    // add total amount of fixed width to subtract from usable space
+    for (let i = 0; i < fixedWidthArray.length; i += 1) {
```
Comment (Chris-Petty) [inline src/js/components/Search.jsx]: Reduce function candidate?

`const totalFixedWidth = fixedWidthArray.reduce((sum, width) => sum += width), 0);`

File: src/js/components/ResultsTable.jsx [hunk 2]
```diff
@@ -5,11 +5,13 @@ import { debounce } from '../utils';
 
 export class ResultsTable extends PureComponent {
   componentDidMount() {
+    // set columns we want to be fixed width
+    const columnWidthData = { 1: 150 };
```
Comment (jtrein) [inline src/js/components/ResultsTable.jsx]: Of course! Thanks for the review. I'll add them to the props on the <ResultsTable columnWidthData={...} />

I'll probably add it to `settings.js` as well, as this is something that will be set infrequently, but it should be configurable from a source when needed to be. This has worked well for me in other projects.

**Comment by jtrein** [comment]
Thanks, this is good to merge, if accepted.

**Pull request merged**